flask_admin/helpers.py

```diff
@@ -131,8 +133,18 @@ def prettify_class_name(name):
 
 
 def is_safe_url(target):
+    # prevent urls like "\\www.google.com"
+    # some browser will change \\ to // (eg: Chrome)
+    # refs https://stackoverflow.com/questions/10438008
+    target = target.replace('\\', '/')
```

Comment:
is there a reason why you're only matching agains double slashes, and not doing `target.replace('\', '/')`?

Reply:
This matches one slash. Your comment is a syntax error because the first slash escapes the second one. You can't put a `\` before a quote at the end of a string literal in python. Try this:
`print '\\``

Reply:
Aah, yes, of course. Thanks @jhatch28